service/openapi.yaml

```diff
@@ -279,3 +314,38 @@ components:
     errorDetails:
       type: string
       description: An error description for logging or other needs
+    metadataResponse:
+      description: OFREP metadata response
+      properties:
+        name:
+          type: string
+          description: name of the flag management system
+          examples:
+            - flagd
+            - go-feature-flag
+        features:
+          type: object
+          description: List of the features supported by the flag management system.
+          properties:
+            cacheInvalidation:
+              $ref: '#/components/schemas/featureCacheInvalidation'
+    featureCacheInvalidation:
+      type: object
+      description: Configuration for the cache cacheInvalidation
+      properties:
+        polling:
+          $ref: '#/components/schemas/featureCacheInvalidationPolling'
+    featureCacheInvalidationPolling:
+      type: object
+      description: Configuration of the polling for the featureCacheInvalidation
+      properties:
+        enabled:
+          type: boolean
+          description: set to true if the remote flag management system is supporting polling
+        minPollingInterval:
+          type: number
+          description: minimum polling interval (in millisecond) supported by the flag management system. The provider should ensure not to set any polling value under this minimum.
+          examples:
+            - 60000
```

Comment:
Where should we specify the defaults? In this case, I would expect polling to be disabled if this is omitted but it would be nice to explicitly state that behavior somewhere.

Reply:
I think I need a better explanation of this feature specifically.

Like if a provider is using a vendor SDK configured with streaming, does that mean it doesn't support polling? Or is the goal here cache freshness value, in which case it would just be however recent you want your flag cache to be. (Correlated to polling because requesting flags greatly faster than the configured polling interval is wasteful.)

Reply:
Wouldn't this be more `configuration` instead of `metadata`?

Reply:
> Nothing prevents vendors to create their own providers and use streaming instead of polling but this will not be offered out of the box by the OFREP providers.

Well not yet right @thomaspoignant? It is really possible that we are doing some server push like SSE in the future, just not yet. And sure, the config only applies to the OFREP providers.

> > Wouldn't this be more configuration instead of metadata?
>
>@weyert Yes, it could be a better name, I wonder what other things about the name?

I also think configuration would be better suited than metadata. The only thing is the name, which might be good for the metadata of the evaluation details, but we could also just decide that this is mapped to them then.

Reply:
>Well not yet right @thomaspoignant? It is really possible that we are doing some server push like SSE in the future, just not yet. And sure, the config only applies to the OFREP providers.

Yes, it will probably come, SSE or even websockets, but this is not part of this 1st version.

>I also think configuration would be better suited than metadata. The only thing is the name, which might be good for the metadata of the evaluation details, but we could also just decide that this is mapped to them then.

I'll change the endpoint to `/ofrep/v1/configuration`.